test/cmd/plugins.sh

```diff
@@ -44,6 +44,10 @@ run_plugins_tests() {
   output_message=$(PATH=${PATH}:"test/fixtures/pkg/kubectl/plugins" kubectl foo)
   kube::test::if_has_string "${output_message}" 'plugin foo'
 
+  # check arguments passed to the plugin
+  output_message=$(PATH=${PATH}:"test/fixtures/pkg/kubectl/plugins/bar" kubectl bar arg1)
+  kube::test::if_has_string "${output_message}" 'kubectl-bar arg1'
```

Comment:
can this verify the first arg is a full path?
```
kube::test::if_has_string "${output_message}" '/test/fixtures/pkg/kubectl/plugins/bar/kubectl-bar arg1'
```

Reply:
Done. Thanks for the review.